packages/flutter/lib/src/widgets/text.dart

```diff
@@ -179,6 +175,25 @@ class DefaultTextStyle extends InheritedWidget {
 /// )
 /// ```
 ///
+/// Using the [Text.rich] constructor, the [Text] widget can
```

Comment:
I believe the `new` is supposed to be before the constructor name here (as in `[new Text.rich]`).